docs/quickstart/installation-guide-for-beginners.mdx

```diff
@@ -0,0 +1,257 @@
+
+    ```
+    kind delete cluster --name odigos-demo
+    ```
+
+    This removes any existing test environment with the same name. If this is your first time, you'll see a message saying the cluster doesn't exist, which is fine.
+
+    b. Now, copy and paste this command and press `Enter`:
+
+    ```
+    kind create cluster --name odigos-demo
+    ```
+
+    This creates a new test environment.
+
+    c. You'll see a lot of text appear in the Terminal. This is normal. Wait until you see a message that says "Your Kubernetes control-plane has initialized successfully!" and the command prompt appears again.
+
+    #### 2. Install the Odigos CLI tool
+
+    The Odigos CLI (Command Line Interface) tool allows you to manage Odigos from the Terminal.
+
+    a. In the Terminal, copy and paste the following command and press `Enter`:
+
+    ```
+    brew install odigos-io/homebrew-odigos-cli/odigos
+    ```
+
+    b. Wait for the installation to complete. You'll know it's done when you see the command prompt again.
+
+    #### 3. Install Odigos in your test environment
+
+    a. In the Terminal, copy and paste the following command and press `Enter`:
+
+    ```
+    odigos install
+    ```
+
+    b. This step may take several minutes. You'll see a lot of text appear in the Terminal – this is normal.
+
+    c. You'll know it's done when you see a message saying "SUCCESS: Odigos installed." and the command prompt appears again.
+
+    #### 4. Open the Odigos control panel
+
+    a. In the Terminal, copy and paste the following command and press `Enter`:
+
+    ```
+    odigos ui
+    ```
+
+    After you run this command, the Terminal will look like it's frozen. This is normal – it's running a program in the background.
+
+    b. Open your web browser (like Safari or Chrome).
+
+    c. In the address bar at the top of the browser window, type exactly this:
+
+    ```
+    http://localhost:3000
+    ```
+
+    Then press `Enter`.
+
+    d. The Odigos control panel should now open in your browser. It might take a few seconds to load.
+
+    #### 5. Set up Odigos
+
+    a. In the Odigos control panel, you'll see a page titled "Select Sources". Sources are the things Odigos can monitor.
+
+    b. Click the checkbox next to each Source you want to monitor. If you're not sure, you can select all of them.
+
+    c. Click the blue "Next" button at the bottom right of the page.
+
+    d. You'll now see a page titled "Add Destinations". Destinations are places where Odigos can send its information.
+
+    e. Click on one of the destination options (for example, "Jaeger").
+
+    f. Fill in any required information. If you're not sure what to enter, consult the Odigos documentation or ask for help.
+
+    g. Click the blue "Create Destination" button.
+
+    h. You should now see an overview page showing your selected Sources and Destinations.
+
+    Congratulations! You've now installed and set up Odigos on your Mac.
```

Comment:
what's next? the reason for installing Odigos is to generate traces and observe the our k8s workloads, but there are no instructions on how to achieve this